gitter/newcomer_messages.py

```diff
@@ -0,0 +1,21 @@
+import logging
+
+import requests
+
+
+def get_newcomer_messages():
+    logger = logging.getLogger(__name__)
+    IMPORT_URL = 'https://pastebin.com/raw/TR4YaaU8'
```

Comment:
Is this for test/demonstration only?

Reply:
Yup, the messages data will come from webservices, after https://gitlab.com/coala/landing/merge_requests/45 is merged.

Those are the output messages from that MR, when I run it locally.